libs/ui/inputs/src/lib/dropdown-multiselect/dropdown-multiselect.component.html

```diff
@@ -43,7 +43,7 @@
   [cdkConnectedOverlayPositions]="overlayPositions"
   [cdkConnectedOverlayScrollStrategy]="scrollStrategy"
   [cdkConnectedOverlayFlexibleDimensions]="true"
-  (backdropClick)="closeOverlay()"
+  (overlayOutsideClick)="closeOverlay()"
```

Comment:
I think there is still a `closeOverlay()` call missing when clicking the select box (with its arrow down) itself.

Reply:
right, thanks!!

Reply:
Thanks, indeed works flawlessly now :-) 

Do I understand correctly that the pointer events on the select box were intercepted by the backdrop that might have been in front of the select box before?

Reply:
I think it's just that the backdrop didn't have any style applied, so it wasn't positioned correctly and capturing anything. In the web components bundle we don't put the full Material theme CSS because it's too big, so we have to extract CSS classes each by each when using a Material component.